uninstall.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+function cleanup() {
+  set +e
+}
+
+function set_variable_from_command() {
+  if [[ ($# -ne 3) ]];
+  then
+    echo "Internal Error - Wrong number of parameters"
+    exit 1
+  fi
+  VAR_NAME=$1
+  DEFAULT_COMMAND=$2
+  if [ -z "${!VAR_NAME}" ]
+  then
+      result=$(bash -c "${!DEFAULT_COMMAND}") || {
+        ret=$?; echo "$3 - Code: $ret"; return $ret;
+      }
+      declare -gx "$VAR_NAME"=$result
+  fi
+  echo "[PRE-UNINSTALL]: $VAR_NAME is set to: ${!VAR_NAME}"
+}
+
+function print_help() {
+   echo "Arguments:"
+   echo "   --help: print this help"
+   echo "   --test: source file only for testing"
+}
+
+function unjoin() {
+  kubectl get clusterconfig configuration -o yaml | sed "s/enableBroadcaster: true/enableBroadcaster: false/g" | kubectl replace -f -
+  sleep 30
```

Comment:
What is this sleep for?

Reply:
Liqo discovery components need maximum 30 seconds to reconcile status across clusters, to have a safe delete of created components/applications

Reply:
This is a quite optimistic assumption, since some component may be stuck and not able to properly reconcile its state. May be we can check that the broadcaster have been really shutoff?